pyscriptjs/src/main.ts

```diff
@@ -158,7 +153,7 @@ export class PyScriptApp {
         // by the try/catch inside main(): that's why we need to .catch() it
         // explicitly and call _handleUserErrorMaybe also there.
         const script = document.createElement('script'); // create a script DOM node
-        script.src = this.interpreter.src;
+        script.src = this.interpreter._remote.src;
```

Comment:
this logic should somehow be moved to the `RemoteInterpreter` eventually, because it is the one who knows how to initialize pyodide/micropython. But I'm fine doing it later